test/smoke/src/helpers/appiumHelper.ts

```diff
@@ -154,4 +169,52 @@ export class appiumHelper {
             return false;
         }, smokeTestsConstants.enableRemoteJSTimeout, `Remote debugging UI element not found after ${smokeTestsConstants.enableRemoteJSTimeout}ms`, 1000);
     }
+
+    private static async openExpoAppViaClipboard(client: WebdriverIO.Client<WebdriverIO.RawResult<null>> & WebdriverIO.RawResult<null>, expoURL: string) {
+        // Expo application automatically detects Expo URLs in the clipboard
+        // So we are copying expoURL to system clipboard and click on the special "Open from Clipboard" UI element
+        console.log(`*** Opening Expo app via clipboard`);
+        console.log(`*** Copying ${expoURL} to system clipboard...`);
+        clipboardy.writeSync(expoURL);
+        console.log(`*** Searching for ${this.EXPO_OPEN_FROM_CLIPBOARD} element for click...`);
```

Comment:
I think it will be better if we move `EXPO_OPEN_FROM_CLIPBOARD` from the class constants to the local variable for consistency with `openExpoAppViaExploreButton` function approach.